### PR TITLE
Issue 52827: Added xfail for pyarrow int64 rounding

### LINKED ISSUE #52827: BUG: int64[pyarrow] throws error when rounded
### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [ ] I have confirmed this bug exists on the [main branch](https://pandas.pydata.org/docs/dev/getting_started/install.html#installing-the-development-version-of-pandas) of pandas.

### Reproducible Example

```python
pd.DataFrame({'x': ['1372636858620000589']}).to_csv('x.csv', index=False)
# this works
pd.read_csv('x.csv', dtype_backend='numpy_nullable')['x'].round()
# this crashes
pd.read_csv('x.csv', dtype_backend='pyarrow')['x'].round()
```

### Issue Description

The Error message is:
```
ArrowInvalid: Integer value 1372636858620000589 not in range: -9007199254740992 to 9007199254740992
```
It seems the value range for `int64[pyarrow]` is too narrow. Shouldn't it be -9223372036854775807 to 9223372036854775807?

### Expected Behavior

be able to manipulate `int64[pyarrow]` just like `Int64`

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : 478d340667831908b5b4bf09a2787a11a14560c9
python           : 3.9.16.final.0
python-bits      : 64
OS               : Darwin
OS-release       : 22.4.0
Version          : Darwin Kernel Version 22.4.0: Mon Mar  6 21:00:17 PST 2023; root:xnu-8796.101.5~3/RELEASE_X86_64
machine          : x86_64
processor        : i386
byteorder        : little
LC_ALL           : None
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 2.0.0
numpy            : 1.24.2
pytz             : 2023.3
dateutil         : 2.8.2
setuptools       : 67.6.1
pip              : 23.0.1
Cython           : None
pytest           : 7.3.1
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : 2.9.6
jinja2           : 3.1.2
IPython          : 8.12.0
pandas_datareader: None
bs4              : 4.12.1
bottleneck       : None
brotli           : None
fastparquet      : 0.8.3
fsspec           : 2023.4.0
gcsfs            : 2023.4.0
matplotlib       : 3.6.3
numba            : None
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : 11.0.0
pyreadstat       : None
pyxlsb           : None
s3fs             : 2023.4.0
scipy            : None
snappy           : None
sqlalchemy       : 2.0.9
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
zstandard        : None
tzdata           : 2023.3
qtpy             : None
pyqt5            : None
</details>

### PR DESCRIPTION
**Pull request by szczekulskij**
- [ ] closes #52827 
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] ~~ Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions. ~~
- [ ]  ~~Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.~~

### CODE REVIEW

File: pandas/tests/frame/methods/test_convert_dtypes.py [hunk 1]
```diff
@@ -134,3 +134,13 @@ def test_pyarrow_engine_lines_false(self):
...
             df.convert_dtypes(dtype_backend="numpy")
+
+    @pytest.mark.xfail(
+        reason="The pyarrow doesn't currentely support rounding of large integers",
+    )
+    def test_pyarrow_dtype_backend_for_int64(self):
+        with tm.ensure_clean("__tmp_to_csv_from_csv3__") as path:
+            # GH 52827
+            pytest.importorskip("pyarrow")
+            pd.DataFrame({"x": ["1372636858620000589"]}).to_csv(path, index=False)
```
Comment (phofl) [inline pandas/tests/frame/methods/test_convert_dtypes.py:145]: Please avoid roundtripping through csv

Comment (szczekulskij) [reply pandas/tests/frame/methods/test_convert_dtypes.py]: Thanks for the feedback.

Updated - checked locally and it throws the right error:
```pyarrow.lib.ArrowInvalid: Integer value 1372636858620000589 not in range: -9007199254740992 to 9007199254740992```

**Comment by szczekulskij** [comment]
Hi @phofl, can I ask for a second review ?

**Comment by mroeschke** [comment]
Feel free to reopen when you have time to circle back here

**Pull request closed without merging**